main.go

```diff
@@ -2,27 +2,44 @@ package main
 
 import (
 	"canopas-website/contact"
+	"canopas-website/db"
```

Comment:
Let's use modules for the directory. We can replace it in go.mod file and use it directly instead of absolute path